libs/category/state/src/effects/category-page.effects.ts

```diff
@@ -70,6 +71,18 @@ export class DaffCategoryPageEffects<
     switchMap((action: DaffCategoryPageLoad) => this.processCategoryGetRequest(action.request)),
   );
 
+  @Effect()
+  loadCategoryPageByUri$: Observable<any> = this.actions$.pipe(
+    ofType<DaffCategoryPageLoadByUri>(DaffCategoryPageActionTypes.CategoryPageLoadByUriAction),
```

Comment:
The action typing should move to line 77

Reply:
moved. I don;t know why `ofType` thinks its generic is a `never`.